example/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.8'
+version: '3.7'
```

Comment:
For the sake of having something contributors can play with, could you add an example implementation to the `example/credentials.tf` file? Anything will do -- it's just there for local experimentation.

Reply:
Added as requested: https://github.com/taiidani/terraform-provider-jenkins/pull/36/commits/1698d7ff7bbf50c74943ffce2bc835a8197a4245